dev/tools/localization/bin/gen_l10n.dart

```diff
@@ -0,0 +1,101 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:io';
+
+import 'package:args/args.dart' as argslib;
+import 'package:file/local.dart' as local;
+import 'package:path/path.dart' as path;
```

Comment:
By default `package:path` will use the current Platform's path separator. If you want to say in sync with the memory filesystem you should either constructor your own path context with the platform you want, or ensure the MemoryFileSystem is using the same platform style